### ISSUE TITLE
Wie Autostart von HAP wenn HAP außerhalb der CCU installiert wird?

### ISSUE THREAD

**Issue by HookHM** [thkl/hap-homematic#666]
Hallo zusammen und Hallo @thkl ,

leider läuft HAP nicht mehr seit der (ich glaube) vorletzten Version von RaspberryMatic. Ich habe hier im Wiki die Option gefunden, dass man HAP auch außerhalb der CCU installieren kann: https://github.com/thkl/hap-homematic/wiki/installation#running-hap-outside-the-ccu

Leider muss man hier HAP mit einem Befehl manuell starten und der Process bleibt in der Konsole geöffnet, beendet man ihn, dann ist HAP auch wieder gestoppt. Wie kann man HAP automatisch starten lassen?

Danke!

**Comment by HookHM** [comment]
Leider nicht. Und wenn ich das Fenster dann nach der Eingabe des Befehls schließe oder per CTRL + C dann ist der Dienst auch beendet.

**Comment by ChrWi** [comment]
Nee, ich meine so wie hier gut beschrieben: https://github.com/thkl/hap-homematic/issues/614#issuecomment-1807680457

**Comment by ChrWi** [comment]
@HookHM: noch was probiert?
